samples/cellular/modem_shell/sample.yaml

```diff
@@ -157,7 +157,6 @@ tests:
     build_only: true
     extra_args: OVERLAY_CONFIG=overlay-modem_fota_full.conf
                 DTC_OVERLAY_FILE="nrf9160dk_ext_flash.overlay"
-                SB_CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY=y
```

Comment:
@nordicjm This option was added to three `modem_shell` configurations in #13153. I don't think these are needed and they do seem to cause a Kconfig warning. Please check.
```
warning: PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY (defined at
/home/tok1/work/ncs/nrf/sysbuild/Kconfig.sysbuild:46) was assigned the value 'y' but got the value
'n'. Check these unsatisfied dependencies: BOOTLOADER_MCUBOOT (=n). See
http://docs.zephyrproject.org/latest/kconfig.html#CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY and/or
look up PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY in the menuconfig/guiconfig interface. The Application
Development Primer, Setting Configuration Values, and Kconfig - Tips and Best Practices sections of
the manual might be helpful too.
```